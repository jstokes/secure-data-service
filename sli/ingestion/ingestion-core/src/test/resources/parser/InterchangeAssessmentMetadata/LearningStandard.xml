<?xml version="1.0" encoding="UTF-8"?>
<InterchangeAssessmentMetadata xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <LearningStandard id="ID001">
    <LearningStandardId ContentStandardName="P7D1FD">
      <IdentificationCode>KkPmmofIfuW1</IdentificationCode>
    </LearningStandardId>
    <Description>l_BDs1R6HA5bfnj</Description>
    <ContentStandard>International Baccalaureate</ContentStandard>
    <GradeLevel>Seventh grade</GradeLevel>
    <SubjectArea>ELA</SubjectArea>
    <CourseTitle>O0j</CourseTitle>
  </LearningStandard>
</InterchangeAssessmentMetadata>
