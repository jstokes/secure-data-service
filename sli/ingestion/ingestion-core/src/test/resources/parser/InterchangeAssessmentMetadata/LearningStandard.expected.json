{
  "a_id":"ID001",
  "LearningStandardId":{
    "a_ContentStandardName":"P7D1FD",
    "IdentificationCode":{"_value":"KkPmmofIfuW1"}
  },
  "Description":{"_value":"l_BDs1R6HA5bfnj"},
  "ContentStandard":{"_value":"International Baccalaureate"},
  "GradeLevel":{"_value":"Seventh grade"},
  "SubjectArea":{"_value":"ELA"},
  "CourseTitle":{"_value":"O0j"}
}